### DISCUSSION TITLE
why not a dockerfile?

### DISCUSSION THREAD

**Discussion by johnhelt** [RandomNinjaAtk/arr-scripts#144] [General]
Hi. First of all, thanks for the work and for sharing it! Really appreciate it. Now my question goes on the method of installation: having to add extra volumes to download several files after the image is created. Why isn't this done as a dockerfile instead, extended the linuxserver ones that already now serves as base images for the extended ones? If possible, this would make installation much more user friendly. I don't know if it is, that's why I'm asking.

**Comment by RandomNinjaAtk** [comment]
Previously a modified container (dockerfile) was provided for easy/simple deployments, but due to monetary expenses related with building images and packaging it that way, I made the decision to not do that anymore...

Example: https://github.com/RandomNinjaAtk/docker-lidarr-extended
